src/Microsoft.DotNet.Build.Tasks/PackageFiles/ApiCompat.targets

```diff
@@ -0,0 +1,134 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="PrereleaseResolveNuGetPackageAssets" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <UsingTask TaskName="LocatePreviousContract" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <PropertyGroup Condition="'$(AssemblyName)' != ''">
+    <AssemblyVersion Condition="'$(AssemblyVersion)'==''">4.0.0.0</AssemblyVersion>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <ApiCompatBaseline>$(MSBuildProjectDirectory)\ApiCompatBaseline.txt</ApiCompatBaseline>
+    <ImplementationOutputPath>$(BaseOutputPath)$(OSGroup).$(Platform).$(ConfigurationGroup)\$(AssemblyName)\$(targetGroup)</ImplementationOutputPath>
```

Comment:
Why not use OutputPath https://github.com/dotnet/corefx/blob/master/dir.props#L386?